apiserver/facades/client/sshclient/facade_test.go

```diff
@@ -208,6 +208,10 @@ func (backend *mockBackend) ModelTag() names.ModelTag {
 	return names.NewModelTag("deadbeef-2f18-4fd2-967d-db9663db7bea")
 }
 
+func (backend *mockBackend) ControllerTag() names.ControllerTag {
+	return names.NewControllerTag("deadbeef-2f18-4fd2-967d-db9663db7bea")
```

Comment:
there's a testing controller tag in `juju/testing`

Reply:
I guess this applies to the model tag too right?

Reply:
Yeah, seems this code is quite old. There's also a testing model tag.